app/containers/GamePage/index.js

```diff
@@ -99,8 +151,22 @@ export default class GamePage extends Component {
               >
                 Send
               </Button>
+              <Button
+                onClick={this.deleteMessages}
+              >
+                Delete messages
+              </Button>
             </FormGroup>
           </form>
+          <Bang
+            assign={assign}
+            database={database}
+            email={email}
+            params={params}
```

Comment:
Instead of passing whole params pass what you need. For example roomId instead of params